### DISCUSSION TITLE
[Feature] use limu to generate immutable json object

### DISCUSSION THREAD

**Discussion by fantasticsoul** [solidjs/solid#2271] [Ideas]
### Describe the bug

[Limu](https://github.com/tnfe/limu) is now the fatest immutable js lib, can solid author consider using it for immutable situation?

### Your Example Website or App

https://tnfe.github.io/limu/

### Steps to Reproduce the Bug or Issue
performance image
![image](https://github.com/user-attachments/assets/395f32b4-7e83-402b-b8a0-6512e93fc41c)

this is a feature.

### Expected behavior

this is a feature.

### Screenshots or Videos

this is a feature.

### Platform

this is a feature.

### Additional context

this is a feature.

**Comment by alex-crabnebula** [comment]
What is an "immutable situation" in the context of solid-js? We rely on signals, not on reconciliation of immutable objects.

**Comment by ryansolid** [comment]
Yeah I'm not sure what this has to do with anything. Maybe I will move this to a discussion.
